BetaCameras/BaslerToF/BaslerToF.cs

```diff
@@ -349,11 +351,11 @@ protected override void UpdateImpl()
                     {
                         intensityImage = new UShortCameraImage(width, height);
 
-                        for (int y = 0; y < height; y++)
+                        for (int y = 0, i = 0; y < height; y++)
                         {
-                            for (int x = 0; x < width; x++)
+                            for (int x = 0; x < width; x++, i++)
                             {
-                                intensityImage[y, x] = bufferIntensity[y * width + x];
+                                intensityImage[y, x] = bufferIntensity[i];
```

Comment:
I'd rename buffertensity to bufferIntensity

Reply:
I don't understand this remark. There is no buffertensity.

Reply:
Hm, it's shown as `buffertensity` on the Github website. I don't find it in the code.